src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -1,70 +1,74 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="add-staff-notation-dialog">
     <v-card>
-      <v-card-title id="dialog-title">
-        <span v-if="administrativeDissolution" id="dialog-title"><strong>{{displayName}}</strong></span>
-        <span v-else-if="putBackOn" id="dialog-title"><strong>Correction - {{displayName}}</strong></span>
-        <span v-else id="dialog-title"><strong>Add a {{displayName}}</strong> </span>
+      <v-card-title>
+        <span id="dialog-title" class="font-weight-bold">
+          <template v-if="isPutBackOn" >Correction - {{displayName}}</template>
+          <template v-else-if="isAdministrativeDissolution">{{displayName}}</template>
+          <template v-else >Add a {{displayName}}</template>
+        </span>
       </v-card-title>
 
       <v-card-text>
-        <div id="dialog-text" class="dialog-text">
-          <p v-if="administrativeDissolution"> You are about to dissolve
-          <strong><span class="text-uppercase">{{getEntityName}}</span>, {{getIdentifier}}</strong> . </p>
-          <p v-if="putBackOn">You are about to put <strong><span class="text-uppercase">{{getEntityName}}</span>,
-          {{getIdentifier}}</strong> back on the register.</p>
-        </div>
-        <div id="notation-text" class="mb-4 mt-2 pt-4">
-          Enter a {{(administrativeDissolution || putBackOn) ? 'Detail' : displayName}}
-          that will appear on the ledger for this entity
-        </div>
+        <p v-if="isAdministrativeDissolution">
+          You are about to dissolve <strong><span class="text-uppercase">{{getEntityName}}</span>,
+          {{getIdentifier}}</strong>.
+        </p>
+        <p v-if="isPutBackOn">
+          You are about to put <strong><span class="text-uppercase">{{getEntityName}}</span>,
+          {{getIdentifier}}</strong> back on the register.
+        </p>
```

Comment:
This is now a paragraph instead of a div.

Old:
![image](https://user-images.githubusercontent.com/10002889/207958368-aae1b298-b3ee-401c-a10e-651e18ca2769.png)

New:
![image](https://user-images.githubusercontent.com/10002889/207958418-6e2ac460-61a5-4f0b-8d26-3a6d6f77aac8.png)